code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -258,3 +258,25 @@
 /turf/closed/wall/r_wall/syndicate/overspace
 	icon_state = "map-overspace"
 	fixed_underlay = list("space"=1)
+
+/////////////////////Pirate Ship walls/////////////////////
+
+/turf/closed/wall/r_wall/syndicate/pirate
+	desc = "Yarr just try to blow this to smithereens!"
+	explosion_block = 30
+	canSmoothWith = list(/turf/closed/wall/r_wall/syndicate/pirate, /obj/machinery/door/airlock/shuttle, /obj/machinery/door/airlock, /obj/structure/window/plastitanium/pirate, /obj/structure/shuttle/engine, /obj/structure/falsewall/plastitanium)
+
+/turf/closed/wall/r_wall/syndicate/pirate/nodiagonal/
+	smooth = SMOOTH_MORE
+	icon_state = "map-shuttle_nd"
+	explosion_block = 30
```

Comment:
Remove, it's already the same as the parent type's

Reply:
Remove, `/turf/closed/wall/r_wall/syndicate/pirate` explosion_block is already 30.